dm/master/config.go

```diff
@@ -294,10 +296,14 @@ func (c *Config) adjust() error {
 
 	if c.PeerUrls == "" {
 		c.PeerUrls = defaultPeerUrls
+	} else {
+		c.PeerUrls = utils.WrapSchemes(c.PeerUrls, c.SSLCA != "")
 	}
 
 	if c.AdvertisePeerUrls == "" {
 		c.AdvertisePeerUrls = c.PeerUrls
+	} else {
+		c.AdvertiseAddr = utils.WrapSchemes(c.AdvertiseAddr, c.SSLCA != "")
```

Comment:
Should handle `AdvertisePeerUrls` rather than `AdvertiseAddr`?

Reply:
> Should handle `AdvertisePeerUrls` rather than `AdvertiseAddr`?

😂 too rely on IDE auto-complete